<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Abhi1996" failures="0" tests="7" name="testCases.TC_03_LoginDDT" time="1.834" errors="1" timestamp="2025-06-07T09:27:59 IST" skipped="6">
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="setup" time="1.834">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:220)
at java.base/java.util.ImmutableCollections$Map1.<init>(ImmutableCollections.java:1112)
at java.base/java.util.Map.of(Map.java:1364)
at org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:176)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:312)
at testBase.BaseClass.setup(BaseClass.java:115)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyLoginDDT -->
</testsuite> <!-- testCases.TC_03_LoginDDT -->
